__testfixtures__/acceptance/nested-in-and-then.input.js

```diff
@@ -8,11 +8,21 @@ test('visiting /twiddles', function(assert) {
     click('.foo');
   });
 
+  andThen(() => {
+    click('.foo');
+  });
+
   andThen(function() {
     andThen(function() {
       andThen(function() {
         click('.foo');
       });
     });
   });
+
+  andThen(() => {
+    assert.ok(true)
```

Comment:
The missing semicolon that is breaking the test comes from transforming this `andThen`, and this is lacking a semicolon as well. Afaik jscodeshift will try to preserve code style, and in this case replace it with the exact same expression, i.e. without semicolon. Adding one here will hopefully also add one into `*.output.js`!